BSP/Simulator/Src/BSP_Timer.c

```diff
@@ -51,10 +48,13 @@ void BSP_Timer_Update()
 
 void BSP_Timer_Init(int Timer1Reload,int Timer2Reload,void *func1,void *func2)
 {
-  FILE *TimerData = fopen("Timer.csv","w");
-  fprintf(TimerData,"%d,%d",Timer1Reload,Timer1Reload);
-  fprintf(TimerData,"\n%d,%d",Timer2Reload,Timer2Reload);
-  fclose(TimerData);
+  FILE *file = fopen("Timer.csv","w");

```

Comment:
The file path here is incorrect. See how the other modules did it by using the macro defined in bsp.h

This incorrect path is causing this to not work since the Python file is updating the correct file and this one isn't.